electron-app/tests/e2e/pages/account-balances-page.ts

```diff
@@ -91,6 +92,45 @@ export class AccountBalancesPage {
     }
   }
 
+  getLocationBalances() {
+    const balanceLocations = [
+      { location: 'Blockchain', renderedValue: Zero },
+      { location: 'Banks', renderedValue: Zero },
+      { location: 'External', renderedValue: Zero },
+      { location: 'Commodities', renderedValue: Zero },
+      { location: 'Real estate', renderedValue: Zero },
+      { location: 'Equities', renderedValue: Zero }
+    ];
+
+    balanceLocations.forEach(balanceLocation => {
+      cy.get('.manual-balances-list tr').then($rows => {
```

Comment:
do we use this `$` prefixing anywhere else? is that a standard in cypress examples?

Reply:
It's a standard in Cypress as per https://docs.cypress.io/guides/core-concepts/variables-and-aliases.html but we don't have to use it. I just used it as I was follow the examples as I learned how to use this. We can change if you want.